File: PRAK402-2410817310014-INDRA_SURYADILAGA.py
bil_maksimal = int(input())
bilangan = 1

while bilangan <= bil_maksimal :
  if bilangan % 2 == 1 :
    print(bilangan, end=" ")
  bilangan = bilangan + 1

print(" ")

while bil_maksimal >= 2 :
  if bil_maksimal % 2 == 0 :
    print(bil_maksimal, end=" ")
  bil_maksimal = bil_maksimal - 1

File: PRAK402-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int bil_maksimal;
    scanf("%d", &bil_maksimal);
    int bilangan = 1;

    while (bilangan <= bil_maksimal) {
        if (bilangan % 2 == 1) {
            printf("%d ", bilangan);
        }
        bilangan++;
    }
    printf("\n");

    while (bil_maksimal >= 2) {
        if (bil_maksimal % 2 ==0) {
            printf("%d ", bil_maksimal);
        }
        bil_maksimal--;
    }
    return 0;
}
